packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -102,6 +103,14 @@ function ListView({
   const fetchClient = useFetchClient();
   const { post, del } = fetchClient;
 
+  const bulkAction = async ({ query, input }) => {
+    const { data } = await post(query, { ...input });
+
+    return data;
+  };
+
+  const bulkPublishMutation = useMutation(bulkAction);
+
```

Comment:
This can go all inline no?

```js
const bulkPublishMutation = useMutation(async ({ query, input }) => {
    const { data } = await post(query, { ...input });

    return data;
  })
```

Also might be nice to write a comment as to what `input` is and `query` im assuming `query` is actually the URL so imo the naming is a bit _odd_

Reply:
Yeah it can go inline, and I can clean up the naming / leave a comment 👍

Reply:
amazing, thanks @markkaylor <3